responsibleai/environments/responsibleai-text-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml

```diff
@@ -26,8 +27,9 @@ dependencies:
     - azureml-dataset-runtime
     - azureml-core
     - --extra-index-url https://azuremlsdktestpypi.azureedge.net/sdk-cli-v2
-    - nlp-feature-extractors==0.0.3
-    - https://publictestdatasets.blob.core.windows.net/packages/pypi/responsibleai-text/responsibleai_text-0.0.7-py3-none-any.whl
+    - nlp-feature-extractors==0.1.0
+    - datasets>=2.9.0
+    - https://publictestdatasets.blob.core.windows.net/packages/pypi/responsibleai-text/responsibleai_text-0.0.13-py3-none-any.whl
```

Comment:
consider to use findlink instead

Reply:
Created a work item on this, will address in the future as all gates are green and ready to check in.

Reply:
[Task 2202691](https://msdata.visualstudio.com/Vienna/_workitems/edit/2202691): curated image update for findlink, pip